salesforce/10.14/modules/ROOT/pages/salesforce-connector-studio.adoc

```diff
@@ -202,6 +202,9 @@ OAuth JWT uses OAuth 2.0 with a JSON Web Token (JWT) request for user authentica
 [[oauth-saml]]
 ==== OAuth SAML Authentication
 
+[NOTE]
```

Comment:
Optional: I don't think the note should be in a table since it is the only entry. In this case, I would just use regular text for this info and not put it in a note.